### PR TITLE
Task1

### PR DESCRIPTION
**Pull request by vanyalaptii**
Was created a Task1

### CODE REVIEW

File: Task1.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    
+    func sumIndexesReturner(_ target: Int) -> [Int] {
+        
+        var result: [Int] = []
+        
+        for firstNumber in nums {
+            let difference = target - firstNumber
+            let secondNumber = nums.firstIndex(of: difference)
+            if secondNumber != nil {
+                result.append(nums.firstIndex(of: firstNumber)!)
```
Comment (MykytaBabanin) [inline Task1.playground/Contents.swift]: remove force unwrap

File: Task1.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+extension Array where Element == Int {
+    
+    func sumIndexesReturner(_ target: Int) -> [Int] {
+        
+        var result: [Int] = []
+        
+        for firstNumber in nums {
+            let difference = target - firstNumber
+            let secondNumber = nums.firstIndex(of: difference)
+            if secondNumber != nil {
```
Comment (MykytaBabanin) [inline Task1.playground/Contents.swift]: use if let construction

File: Task1.playground/Contents.swift (hunk 1 above)
Comment (vanyalaptii) [reply Task1.playground/Contents.swift]: Done

File: Task1.playground/Contents.swift (hunk 2 above)
Comment (vanyalaptii) [reply Task1.playground/Contents.swift]: Done

**Review by MykytaBabanin — APPROVED** [review]

**Pull request merged**